Experimenter/Experimenter.UI/ApplicationMain.cs

```diff
@@ -6,13 +6,15 @@
 
 namespace DistributedExperimentation.Experimenter.UI
 {
-    class ApplicationMain
+    public class ApplicationMain
     {
-        static void Main(string[] args)
+        public static void Main(string[] args)
         {
             try {
                 Dictionary<string, string> parsedArgs = parseArguments(args);

```

Comment:
I think you did interpret this wrong. 
if you pass 
*.exe --parmeter1 valueParameter1 --parameter2 parameter2  
into Commandline you end up with 4 args. but you parse it as it where 2 arguments and have to split them manualy.

Reply:
You are right. I have been fixed this